.gitignore

```diff
@@ -27,10 +27,11 @@ object_script.*.Debug
 *_plugin_import.cpp
 *.moc
 ui_*.h
-*.html
+*html
```

Comment:
Why `*html` instead of `*.html`? (What kind of additional files is this filtering out that the old one didn't catch?)

Reply:
because there are directories made by squish that are scopecoverage_html and I would like to ignore them too

Reply:
:+1: Thank you!